lib/beaker/platform.rb

```diff
@@ -13,7 +13,8 @@ class Platform < String
                      "wheezy"    => "7",
                      "squeeze"   => "6",
                    },
-        :ubuntu => { "focal"   => "2004",
+        :ubuntu => { "jazzy"   => "2204",
```

Comment:
I think 22.04 is codenamed "jammy"

Reply:
Ugh, late night PR, what could go wrong? Thanks for catching this.